frontend/src/components/2DModel/Modelization.vue

```diff
@@ -495,7 +505,7 @@ export default {
 				d3.select(this).transition().attr("fill", "black");
 				let terraformObject = new TerraformObjectNode(
 					panelObject,
-					"myTerraformObjectNode",
+					`myTerraformObjectNode#${randomHexString(4)}`,
```

Comment:
not use `myTerraformObjectNode` but use the type of the component, like

aws_instance_HASH :)